lib/src/widgets/chat/ds_video_message_bubble.widget.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/widgets/buttons/ds_play_button_rounded.widget.dart';
+
+class DSVideoMessageBubble extends StatelessWidget {
+  /// Aligns the card to the right or left according to the value assigned to [align] which can be [DSAlign.right] or [DSAlign.left].
+  final DSAlign align;
+
+  /// URL of the video that will be played when clicking on the card
+  final String url;
+
+  final List<DSBorderRadius> borderRadius;
```

Comment:
You could add some doc comments here as well.

Reply:
Done